docs/extras/save_management.md

```diff
@@ -1,19 +1,19 @@
 # Save Management
 
-For save management,  [Edizon](https://github.com/WerWolv/EdiZon/releases) is recommended. It can be used to back up and restore game saves to your SD card. It also has the ability to edit saves for extra items etc. It also has support for Atmosphere's cheat engine, providing an easy way to download new cheats, as well as toggle them on or off.
+For save management, [JKSV](https://github.com/J-D-K/JKSV) is recommended. It can be used to back up and restore game saves to your SD card. 
 
-[Checkpoint](https://github.com/flagbrew/checkpoint/releases) is also a save manager. It can be used to back up and restore game saves to your SD card. It also has the ability to share save data over FTP and WiFi.
+[Checkpoint](https://github.com/SciresM/Checkpoint/releases/tag/ams-19-compat) is also a save manager. It can be used to back up and restore game saves to your SD card. It also has the ability to share save data over FTP and WiFi.
 
 &nbsp;
 	
-### Edizon
-Please read Edizon's [homepage](https://github.com/WerWolv98/EdiZon) for information on how to use it.
+### JKSV
+Please read JKSV's [homepage](https://github.com/J-D-K/JKSV) for information on how to use it.
 
 !!! tip ""
-	![ExampleSwitchTheme](../extras/img/save_edizon.jpg)
+	![ExampleSwitchTheme](../extras/img/save_jksv.jpg)
 
 ### Checkpoint
-Please read Checkpoint's [usage page](https://github.com/FlagBrew/Checkpoint#usage) for information on how to use it.
+Please read Checkpoint's [homepage](https://github.com/SciresM/Checkpoint#usage) for information on how to use it.
```

Comment:
Probably want to change the link to "https://github.com/SciresM/Checkpoint" to match with the "homepage" name